.github/workflows/ci.yaml

```diff
@@ -84,7 +85,7 @@ jobs:
           provider: microk8s
           channel: 1.28-strict/stable
           microk8s-group: snap_microk8s
-          microk8s-addons: "hostpath-storage dns minio metallb:${{ env.IPADDR }}-${{ env.IPADDR }}"
+          microk8s-addons: "rbac hostpath-storage dns minio metallb:${{ env.IPADDR }}-${{ env.IPADDR }}"
```

Comment:
Here, I've enabled `rbac` in our test K8s environment because without RBAC, there's no point testing cluster permissions.

Reply:
Indeed! Good catch! I believe we  should always test with RBAC enabled! Thanks!